### ISSUE TITLE
role-epel missing

### ISSUE THREAD

**Issue by vchuravy** [staticfloat/julia-ansible-scripts#3]
I am trying to setup vagrant in order to reproduce issues happening on the buildbots. (x-ref https://github.com/JuliaLang/julia/pull/18734#issuecomment-254481230)

Instead of setting of tyrying to setup the same environment by hand I was hoping that I could use ansible to do it for me, but I am running into the following.

```
ERROR! the role 'repo-epel' was not found in /home/wallnuss/src/julia-ansible-scripts/roles:/home/wallnuss/src/julia-ansible-scripts:/etc/ansible/roles

The error appears to have been in '/home/wallnuss/src/julia-ansible-scripts/buildslaves.yml': line 20, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
    - repo-epel
```

cc: @staticfloat

**Comment by staticfloat** [comment]
Yes, there are many roles that you will need to clone down.  If you [search my repository list for `role`](https://github.com/staticfloat?page=1&q=role+epel&tab=repositories&utf8=%E2%9C%93&utf8=%E2%9C%93&q=role), you can find the [`repo-epel` repository](https://github.com/staticfloat/ansible-role-repo-epel).  Clone that down into a directory called `repo-epel` in `/home/wallnuss/src/julia-ansible-scripts/roles`.  You'll probably also need the [`julia_buildslave` role](https://github.com/staticfloat/ansible-julia_buildslave), and [`startup_script`](https://github.com/staticfloat/ansible-role-startup_script).
